pkg/annotation/k8s.go

```diff
@@ -37,6 +37,9 @@ const (
 	// CNIPrefix is the common prefix for CNI related annotations.
 	CNIPrefix = "cni.cilium.io"
 
+	// TaintPrefix is the common prefix for taint annotations.
+	TaintPrefix = "taint.cilium.io"
```

Comment:
This is a neat facility, though I am concerned the use of the name "taint", given that we commonly operate in k8s environments and kubernetes already has a taints concept that impacts scheduling: https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/ . Perhaps using something like `status.cilium.io` or reusing `config.cilium.io` could be viable alternatives?

Reply:
Yep makes sense. I'll use a different prefix and different header in cilium status to avoid the confusion.